TestFoundation/TestNSURLSession.swift

```diff
@@ -46,6 +48,8 @@ class TestURLSession : XCTestCase {
     
     override class func setUp() {
         super.setUp()
+
+        _ = signal(SIGPIPE, SIG_IGN)
```

Comment:
we might want to add this earlier in the lifecycle (e.g. main.swift) so that it applies equally to all tests (perhaps we might add more in the future that will need this